src/fu-engine.c

```diff
@@ -6287,7 +6287,7 @@ fu_engine_load (FuEngine *self, FuEngineLoadFlags flags, GError **error)
 	FuQuirksLoadFlags quirks_flags = FU_QUIRKS_LOAD_FLAG_NONE;
 	g_autoptr(GPtrArray) checksums_approved = NULL;
 	g_autoptr(GPtrArray) checksums_blocked = NULL;
-#ifndef _WIN32
+#ifndef __linux__
```

Comment:
shouldn't this be `#ifdef`  not `#ifndef`

Reply:
Fixed.